packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -28,26 +28,6 @@ export interface TableColumnResizeAria {
   resizerProps: DOMAttributes
 }
 
-export interface AriaTableColumnResizeProps<T> {
-  /** An object representing the [column header](https://www.w3.org/TR/wai-aria-1.1/#columnheader). Contains all the relevant information that makes up the column header. */
-  column: GridNode<T>,
-  /** Aria label for the hidden input. Gets read when resizing. */
-  label: string,
-  /**
-   * Ref to the trigger if resizing was started from a column header menu. If it's provided,
-   * focus will be returned there when resizing is done.
-   * */
-  triggerRef?: RefObject<HTMLDivElement>,
-  /** If resizing is disabled. */
-  isDisabled?: boolean,
-  /** Called when resizing starts. */
-  onResizeStart?: (widths: Map<Key, number | string>) => void,
-  /** Called for every resize event that results in new column sizes. */
-  onResize?: (widths: Map<Key, number | string>) => void,
-  /** Called when resizing ends. */
-  onResizeEnd?: (widths: Map<Key, number | string>) => void
-}
-
 
 export interface TableLayoutState {
```

Comment:
Kinda unrelated, but why is this an interface here rather than in stately? Looks like there's an almost identical one there.

Reply:
Looks like I could move it to the stately/table package and then use it in the stately/layout package
At one point I'd had a circular dependency with this since both the hook and the class needed to implement this interface